can-map-define_test.js

```diff
@@ -1565,3 +1571,62 @@ QUnit.test("can.hasKey", function() {
 	vm.attr('lateProp', 'something');
 	equal(canReflect.hasKey(vm, "lateProp"), true, "vm.hasKey('lateProp') true");
 });
+
+QUnit.test("can.getOwnEnumerableKeys", function() {
+	var ParentMap = CanMap.extend({
+		define: {
+			parentNoEnum: {
+				serialize: false,
+				value: 'parent_no'
+			},
+
+			parentEnum: {
+				serialize: true,
+				value: 'parent_yes'
+			},
+
+			parentEnumByDefault: {
+				value: 'maybe'
+			},
+
+			parentEnumGetter: {
+				get: function () {
+					return 'parent_get';
+				}
+			}
+		}
+	});
+
+		var VM = ParentMap.extend({
```

Comment:
Something seems up with the spacing here.

Reply:
Should be good now